tracer/test/test-applications/security/Samples.Security.AspNetCore5/Views/Home/Index.cshtml

```diff
@@ -49,6 +49,11 @@
 
     <div>&nbsp;</div>
 
+    <div> <a href="/EnableASM">Enable ASM</a></div>
```

Comment:
does this match any controller route? 🤔

Reply:
I think this left over from an old way of testing activation, could be removed, but not that important.

Reply:
Fixed in 075b87c775ee31862f472ebbb526303a41224d25